cds/modules/flows/files.py

```diff
@@ -86,6 +86,7 @@ def dispose_object_version(object_version):
     """Clean up resources related to an ObjectVersion."""
     object_version = as_object_version(object_version)
     # remove the object version
+    object_version.bucket.locked = False
```

Comment:
Do we have to relock it after deletion if it was locked before?

Reply:
Since we are deleting I thought it doens't make any difference, but you are right it might be that we only use it to delete partially some objectVersions, so we should leave it in the same state. Thanks!